### ISSUE TITLE
CocoaPods GeoFire Dependency Outdated

### ISSUE THREAD

**Issue by mleifer6** [firebase/geofire-objc#127]
It looks like the CocoaPods spec file for GeoFire is out of date as it list the wrong version of Firebase/Database as a dependency. 

As you can see [here](https://github.com/firebase/geofire-objc/blob/master/GeoFire.podspec) (https://github.com/firebase/geofire-objc/blob/master/GeoFire.podspec), GeoFire depends on v6.0. However, [on CocoaPods/Specs for GeoFire](https://github.com/CocoaPods/Specs/blob/9543734012238c33ea018af5a4381459c13f0916/Specs/2/8/4/GeoFire/3.0.0/GeoFire.podspec.json) (https://github.com/CocoaPods/Specs/blob/9543734012238c33ea018af5a4381459c13f0916/Specs/2/8/4/GeoFire/3.0.0/GeoFire.podspec.json), v5.0 is listed as a dependency.

**Comment by scm573** [comment]
Also I found there's no v3.0.0 release published in this repo.
So currently we have to specify the branch
```ruby
pod 'GeoFire', :git => 'https://github.com/firebase/geofire-objc.git', :branch => 'master'
```

If possible, please publish the release and fix the CocoaPods distribution. Appreciate.

**Comment by hanawat** [comment]
> If possible, please publish the release and fix the CocoaPods distribution.

+1

**Comment by chrispypatt** [comment]
Is this something that will be published? I'm working on a project that requires GeoFire as a dependency of another pod's pubspec and because of this, I cannot update other Firebase tools.

**Comment by brokenseal** [comment]
Yes, please, show some love to this repo! 👍

**Comment by brokenseal** [comment]
Is this repository still maintained? Should I fork it? I would prefer not to, honestly...

**Comment by morganchen12** [comment]
Hey all, I'll update the podspec with a new release this week.

**Comment by morganchen12** [comment]
Hey all, took a little over a week, but this is released in GeoFire 4.0.1 now. Please let me know if there are any issues.

**Comment by vaibhav-jhaveri-ds** [comment]
@morganchen12 What is the status on #64?
